### PR TITLE
Editorial pass through latter half

### PR DESCRIPTION
**Pull request by rhiaro**
Removed some redundant language and consistently structured DID documents and DID methods sections. Also a few untestable normative statements in there have been rephrased or removed.

Removed (rephrased) some normative language in Security and Privacy considerations sections.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/108.html" title="Last updated on Nov 25, 2019, 4:10 PM UTC (ed39181)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/108/a8d0c28...ed39181.html" title="Last updated on Nov 25, 2019, 4:10 PM UTC (ed39181)">Diff</a>

### CODE REVIEW

**Comment by rhiaro** [comment]
If I dropped some MAYs from normative sections it seemed like they were really informative as they wouldn't have had an effect on interop, which makes normative language potentially more confusing, eg. 

```diff
- DID method specifications MAY create intermediate representations of 
+ DID method specifications can create intermediate representations of
a DID document [during the process of resolution]
``` 

but of course happy to revise if I got any of that wrong.

**Comment by msporny** [comment]
> If I dropped some MAYs from normative sections it seemed like they were really informative as they wouldn't have had an effect on interop, which makes normative language potentially more confusing

I think you did the right thing, just waiting to see if anyone disagrees.

File: index.html [hunk 1]
```diff
@@ -1712,29 +1661,25 @@ <h2>
...
 
-        <li>
-The value of this key MUST follow the formatting rules (3, 4, 5)
-from Section <a href="#created-optional"></a>.
-        </li>
-      </ol>
+      <dl>
+          <dt><dfn>updated</dfn></dt>
+          <dd>
+The value <code>updated</code> MUST follow the same formatting rules as the
```
Comment (TallTed) [inline index.html]:
```suggestion
The <code>updated</code> value MUST follow the same formatting rules as the
```

File: index.html [hunk 2]
```diff
@@ -2125,40 +2064,19 @@ <h2>
...
+required of a CKMS (cryptographic key management system), e.g.: key registration,
+key replacement, key rotation, key recovery, and key expiration.
 
-        <li>
-Key expiration
-        </li>
-      </ul>
+      </p>
+      <p>
+Any <a>DID method</a> specification that does not support specific operations,
```
Comment (TallTed) [inline index.html]:
```suggestion
Any <a>DID method</a> specification that does not support a specific operation,
```

File: index.html [hunk 3]
```diff
@@ -2125,40 +2064,19 @@ <h2>
...
+such as Update and Deactivate, MUST clearly state these limitations.
```
Comment (TallTed) [inline index.html]:
```suggestion
such as Update or Deactivate, MUST clearly state these limitations.
```

File: index.html [hunk 4]
```diff
@@ -2203,8 +2121,7 @@ <h3>
         </h3>
 
         <p>
-Although a core feature of <a>distributed ledgers</a> is immutability, the
-<a>DID method</a> specification MUST specify how a client can deactivate a
+The <a>DID method</a> specification MUST specify how a client can deactivate a
 <a>DID</a> on the <a>DID registry</a>, including all cryptographic
 operations necessary to establish proof of deactivation <em>or</em> state that
```
Comment (TallTed) [inline index.html]:
```suggestion
operations necessary to establish proof of deactivation, <em>or</em> state that
```

**Comment by rhiaro** [comment]
Thanks @TallTed

**Pull request merged**